mpbb-install-port

```diff
@@ -43,6 +43,15 @@ install-port() {
     #> "$log_port_contents"
     > "$log_port_stats"
 
+    # $option_prefix and $thisdir are set in mpbb
+    # shellcheck disable=SC2154
+    if [[ -f $("${option_prefix}/bin/port-tclsh" "${thisdir}/tools/archive-path.tcl" "$@") ]]; then
+        echo "$@ already installed, nothing to do"
+        # log: summary for the portwatcher
+        echo "Building '$port' ... [OK]" >> "$log_subports_progress"
```

Comment:
I would probably prefer `... [SKIP]` to `[OK]`. But as mentioned earlier, I would put this to the part that decides which dependencies to build. (The code could potentially remain here as a safeguard anyway.)